### ISSUE TITLE
Save model with best validation accuracy on Sentiment Analysis with RNN

### ISSUE THREAD

**Issue by amitupreti** [udacity/deep-learning-v2-pytorch#215]
In case of this project. 
https://github.com/udacity/deep-learning-v2-pytorch/blob/master/sentiment-rnn/Sentiment_RNN_Solution.ipynb

Why aren't we saving the model with best validation score and using it for testing and prediction?

**Comment by abhiojha8** [comment]
Though we are a little late to respond, we encourage such _user-specific_ issues to be raised on the Knowledge hub, so that our experienced mentors can chime-in and new students can also learn from the discussion thread. 
Thank you
